app/helpers/maintenance_tasks/tasks_helper.rb

```diff
@@ -101,16 +101,38 @@ def csv_file_download_path(run)
       )
     end
 
-    # Return the appropriate field tag for the parameter, based on its type.
-    # If the parameter has a `validates_inclusion_of` validator, return a dropdown list of options instead.
-    def parameter_field(form_builder, parameter_name)
-      inclusion_validator = form_builder.object.class.validators_on(parameter_name).find do |validator|
+    # Returns the appropriate value for validator's inclusion options for a Proc,
+    # lambda, symbol, or other types.
+    #
+    # If there is a value returned, the dropdown list of options will be used.
+    #
+    # Procs, lambdas, and symbols options are not supported by the dropdown form,
+    # and return nil.
+    #
+    # @param object [Task] The object for which the value needs to be resolved.
+    # @param parameter_name [String] The parameter name.
+    #
+    # @return [Object] value of the resolved inclusion option.
```

Comment:
Suggestion to condense / clarify the language a bit:

```suggestion
    # Resolves values covered by the inclusion validator for a task attribute.
    # Procs, lambdas, and symbols options are not supported and return nil.
    #
    # Returned values are used to populate a dropdown list of options.
    #
    # @param object [Task] The object for which the value needs to be resolved.
    # @param parameter_name [String] The parameter name.
    #
    # @return [Enumerable] value of the resolved inclusion option.
```

Reply:
Also, what about passing `object.class` to this method instead? Then we could call the parameter `task_class` or `task`, which is a bit easier to gather meaning from than `object`.

```
# @param task [Class<Task>] The task whose inclusion validator needs to be resolved.
```

Reply:
> Also, what about passing object.class to this method instead? 

That's a good idea -- applied in https://github.com/Shopify/maintenance_tasks/pull/962/commits/af4e6b0ec4affe1f529043ec9ed93e0bb5f1167c